appformer-kogito-api/src/main/java/org/appformer/kogito/api/stateControl/registry/DefaultCommandRegistry.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.appformer.kogito.api.stateControl.registry;
+
+public interface DefaultCommandRegistry<C> extends CommandRegistry<C> {
```

Comment:
Do we need this? Can't `DefaultCommandRegistryImpl` implement directly `CommandRegistry`?

Reply:
The  reason of doing it like that is that stunner is using the CommandRegistry in different places and I had to differenciate it by adding that extra interface to make. In regular CDI This could be achieved by using qualifiers for example but this approach was breaking the GWT compilation. So I had to do it like that.